spring-integration-cassandra/src/main/java/org/springframework/integration/cassandra/config/xml/CassandraOutboundChannelAdapterParser.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2015 the original author or authors.
```

Comment:
Yeah... We are in a new year already.
Since it isn't in master yet, we should update Copyrights of affected classes to the current year. :smile: